File: Lemur/src/frc/team4330/robot/server/ServerProtocol.java
package frc.team4330.robot.server;

import frc.team4330.robot.Robot;

public class ServerProtocol {

	public String processDataRequests(String requestSeg) {
		String output;

		if (requestSeg.equalsIgnoreCase("DATA"))
			output = "[" + Robot.gyro.getVelocityX() + ", " 
					+ Robot.gyro.getVelocityY() + ", " 
					+ Robot.gyro.getAngle() + "]";
		else
			output = null;


		return output;
	}
}

File: Lemur/src/main/java/frc/team4330/robot/server/ServerProtocol.kt
package frc.team4330.robot.server

import frc.team4330.robot.Robot

class ServerProtocol {

    fun processDataRequests(requestSeg: String): String? {
        val output: String?

        if (requestSeg.equals("DATA", ignoreCase = true))
            output = ("[" + Robot.gyro.velocityX + ", "
                    + Robot.gyro.velocityY + ", "
                    + Robot.gyro.angle + ", "
                    + Robot.gyro.rate + "]")
        else
            output = null


        return output
    }
}
